contracts/multisig-prover/src/msg.rs

```diff
@@ -15,6 +15,11 @@ pub enum ExecuteMsg {
     // Queries the gateway for actual message contents
     #[permission(Any)]
     ConstructProof(Vec<CrossChainId>),
```

Comment:
Is this method useful at all for the stacks integration? Do we always need the payload? If so, I think this method should be removed

Reply:
To reduce risk of bugs from two ways of constructing proofs for the same ITS msg, I'd suggest removing this as well, until we add ITS hub translation support, and then all proofs can be constructed via this approach